worldguard-core/src/main/java/com/sk89q/worldguard/commands/region/RegionCommands.java

```diff
@@ -475,6 +479,16 @@ public void list(CommandContext args, Actor sender) throws CommandException {
             task.filterOwnedByName(ownedBy, args.hasFlag('n'));
         }
 
+        if (args.hasFlag('s')) {
+            ProtectedRegion existing = checkRegionFromSelection(sender, "tmp");
+            task.filterByIntersecting(existing);
+        }
+
+        // -i string is in region id
+        if (args.hasFlag('i')) {
+            task.filterIdByMatch(idFilter = args.getFlag('i'));
```

Comment:
`idFilter` is assigned here but not used anywhere? could just drop the local entirely?